firefox-ios/firefox-ios-tests/Tests/L10nSnapshotTests/L10nSuite1SnapshotTests.swift

```diff
@@ -113,6 +113,7 @@ class L10nSuite1SnapshotTests: L10nBaseSnapshotTests {
     @MainActor
     func testTopSitesMenu() {
         sleep(3)
+        waitForTabsButton()
```

Comment:
Sometimes, `testTopSitesMenu()` fails because the app loads slowly. The test times out before the `NewTabScreen` is shown completely.